docs/modules/general/pages/local-env.adoc

```diff
@@ -0,0 +1,27 @@
+= OneCX Local Environment
+:local-env-url: https://github.com/onecx/onecx-local-env?tab=readme-ov-file#onecx-local-environment-setup
+
+== Overview
+With the OneCX platform, every Application needs to adjusted so it can be integrated within the platform. For the best experience with Application development and integration, OneCX offers a {local-env-url}[OneCX Local Enviornment] repository. It provides a consistent setup of the platform and Core Applications with minimal actions and allows extension.
```

Comment:
suggestion: With the OneCX platform, every application needs to be adjusted for integration within the platform.